components/eamxx/src/share/field/field_tracking.cpp

```diff
@@ -31,4 +31,10 @@ void FieldTracking::update_time_stamp (const TimeStamp& ts) {
   }
 }
 
+void FieldTracking::set_accum_start_time (const TimeStamp& t_start) {
+  EKAT_REQUIRE_MSG (not m_time_stamp.is_valid() || m_time_stamp<=t_start,
```

Comment:
will this be **not** "m_time_stamp.is_valid() or m_time_stamp<=t_start" or will it be "not m_time_stamp_is_valid()" **or** "m_time_stamp<=t_start"?  In other words, what is the `not` effecting?  I'm guessing the former, but wanted to make sure.